src/classes/HiFiMixerSession.ts

```diff
@@ -897,6 +914,9 @@ export class HiFiMixerSession {
                 }
                 this._setCurrentHiFiConnectionState(HiFiConnectionStates.Failed);
                 break;
+            case RaviSessionStates.CLOSED:
+               this._setCurrentHiFiConnectionState(HiFiConnectionStates.Disconnected);
+               break;
```

Comment:
This is a functional-change / bug-fix.  Without this applications (and my code) never gets to learn that we closed!

Reply:
This change is in tip of main as of https://github.com/highfidelity/hifi-spatial-audio-js/pull/129 if you want to pull that in.